paywall/src/middlewares/currencyConversionMiddleware.js

```diff
@@ -13,18 +17,28 @@ export default store => {
         setConversionRate(info.data.data.currency, info.data.data.amount)
       )
     )
-  setInterval(() => {
```

Comment:
Note that the way `pollWithConditions` is not thru `setInterval`. This may not be a big deal, but I wonder about un-anticipated side-effects here.